examples/paths.py

```diff
@@ -0,0 +1,5 @@
+import os
+
+default_cf_mapping_files = [os.path.dirname(__file__) + '/cloudformation/cloudformation_map.yaml',
```

Comment:
These default set-up files should be able to be provided by the users when launching the server, instead of relying on them being in one of our inner folders and having the user "guess" or investigate what they have to edit if they need to perform changes.